pom.xml

```diff
@@ -393,6 +395,7 @@
               <exclude>src/ports/greenplum/cmake/FindGreenplum_4_2.cmake</exclude>
               <exclude>src/ports/greenplum/cmake/FindGreenplum_4_3.cmake</exclude>
               <exclude>src/ports/greenplum/cmake/FindGreenplum_4_3ORCA.cmake</exclude>
+              <exclude>src/ports/greenplum/cmake/FindGreenplum_5.cmake</exclude>
```

Comment:
Since we do have a license header for this file, there is no need to exclude it.